boards/nxp/mimxrt685_evk/mimxrt685_evk_mimxrt685s_cm33.dts

```diff
@@ -259,7 +259,7 @@ i2s1: &flexcomm3 {
 		status = "okay";
 		jedec-id = [c2 81 3a];
 		erase-block-size = <4096>;
-		write-block-size = <16>;
+		write-block-size = <1>; /* FLASH_MCUX_FLEXSPI_MX25UM51345G_OPI_STR set */
```

Comment:
Any idea why we use OPI_STR on the RT685? The `git blame` doesn't provide a good reason (a4c9e13ea8ddcd8652). Probably out of scope for this PR, but maybe we should enable OPI_DTR mode, I believe it will provide better throughput.

Reply:
@danieldegrasse, per @ChayGuo.
The flash on RT600 EVK is connected to Flexspi PORTB, and per the DS, portB only has SDR mode.